frontend/libs/erxes-ui/src/modules/hotkey/components/PopoverScoped.tsx

```
@@ -0,0 +1,29 @@
+    goBackToPreviousHotkeyScope,
+  } = usePreviousHotkeyScope();
+
+  return (
+    <Popover
+      {...props}
+      onOpenChange={(open) => {
+        onOpenChange?.(open);
+        if (scope) {
+          open
+            ? setHotkeyScopeAndMemorizePreviousScope(scope + '.Popover')
+            : goBackToPreviousHotkeyScope();
```

ESLint

[error] Expected an assignment or function call and instead saw an expression.

(@typescript-eslint/no-unused-expressions)